subprojects/file-watching/src/test/groovy/org/gradle/internal/watch/registry/impl/AbstractFileWatcherUpdaterTest.groovy

```diff
@@ -129,7 +129,8 @@ abstract class AbstractFileWatcherUpdaterTest extends Specification {
     }
 
     static RegularFileSnapshot snapshotRegularFile(File regularFile) {
-        new RegularFileSnapshot(regularFile.absolutePath, regularFile.name, TestFiles.fileHasher().hash(regularFile), FileMetadata.from(Files.readAttributes(regularFile.toPath(), BasicFileAttributes)), AccessType.DIRECT)
+        def attributes = Files.readAttributes(regularFile.toPath(), BasicFileAttributes)
+        new RegularFileSnapshot(regularFile.absolutePath, regularFile.name, TestFiles.fileHasher().hash(regularFile), DefaultFileMetadataSnapshot.file(attributes.lastModifiedTime().toMillis(), attributes.size(), AccessType.DIRECT))
```

Comment:
Nit: it would be in general a good idea to split long lines like this. Here specifically extracting the `FileMetadata(Snapshot)` as a local variable, or put it on a new line would make a lot of sense to me.

Reply:
I added some newlines.